### ISSUE TITLE
Store Reconstruction Parameters

### ISSUE THREAD

**Issue by tknopp** [MagneticParticleImaging/MDF#46]
I think it would be good to store the reconstruction parameters in the `reconstruction` group. I think about putting them into a directory `/reconstruction/parameters`

Here is a list of our most important reconstruction parameters in Julia Dictionary syntax:

```
  params[:lambd] = 1e-2
  params[:iterations] = 4
  params[:SNRThresh] = 2.0
  params[:minFreq] = 80e3
  params[:maxFreq] = 1.25e6
  params[:sortBySNR] = false
  params[:nAverages] = 1
  params[:repetitionTime] = 0.0
  params[:denoiseWeight] = 0
  params[:loadas32bit] = true
  params[:loadasreal] = false
  params[:sparseTrafo] = nothing
  params[:redFactor] = 0.0
  params[:solver] = "kaczmarz"
  params[:emptyMeasPath] = nothing
  params[:SFPath] = nothing
  params[:frames] = 1
  params[:spectralCleaning] = true
  params[:recChannels] = [1,2,3]
```

Naming is certainly not perfect/consistent. Finding a standard set of parameters might be challenging, because the reconstruction is always a little in flux due to new developments. There are also difficult dependencies (`iterations` for `SVD` for instance)

Ping @MandyA @profix898 @hofmannmartin

**Comment by hofmannmartin** [comment]
Since there is no standard MPI reconstruction library it might be sufficient to add an optional subgroup to reconstruction which contains the parameters.

I think at this stage it is completely sufficient if every group uses their own parameters, since in general they are the only once to be able to perform the reconstruction any way.

In my opinion a common set of parameters only makes sense if a standard reconstruction library is available.

**Comment by MandyA** [comment]
I agree with @hofmannmartin. Currently, a common set of paramters is hardly possible. But I like the idea of storing all necessary paramters (per research group) in an optional reconstruction group.

**Comment by tknopp** [comment]
Yes, might to early for this. We need to do this internally here though and might come up with a proposal. Its also linked to #45 because it does not make to much sense storing different reconstructions of the same dataset when the reconstruction parameters are not stored.

I just want to open this issue early so that everybody interested can take part in that discussion.

**Comment by tknopp** [comment]
We have basically implemented this and are do store the used reconstruction parameters.
